src/HealthChecks.AzureStorage/DependencyInjection/AzureStorageHealthCheckBuilderExtensions.cs

```diff
@@ -82,5 +84,30 @@ public static IHealthChecksBuilder AddAzureQueueStorage(this IHealthChecksBuilde
                tags,
                timeout));
         }
+
+        /// <summary>
+        /// Add a health check for Azure Queue Storage.
+        /// </summary>
+        /// <param name="builder">The <see cref="IHealthChecksBuilder"/>.</param>
+        /// <param name="queueServiceUri">The Azure Queue service Uri (https://myaccount.blob.core.windows.net)). </param>
+        /// <param name="credential">The TokenCredentail to use, you can use Azure.Identity with DefaultAzure Credential or other kind of credentials,you can read more on https://github.com/Azure/azure-sdk-for-net/blob/Azure.Identity_1.2.2/sdk/identity/Azure.Identity/README.md. </param>

```

Comment:
You can use `<see href="url-here"/>` to encode the URL so it's clickable in Visual Studio and rendered in any HTML docs (depending on the generator, but most seem to do it).

Reply:
True, also my fault.. I write the code in hurry :-(